frontend/src/components/predictiontoolsobjs/LineChart.jsx

```diff
@@ -0,0 +1,196 @@
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+import { Line } from "react-chartjs-2";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend
+);
+// above imports all using Chart-js library!
+
+// dummay data, mostly just to understand how to format my TC #2 data!
+const dummyData = [
```

Comment:
Not very important, but could just put this test data in a separate file, so it's easier to clean up later.